[18:45:21] I/launcher - Running 1 instances of WebDriver
[18:45:21] I/hosted - Using the selenium server at http://localhost:9515
Spec started
A Jasmine spec timed out. Resetting the WebDriver Control Flow.

  Protractor sync test cases
    [31mâœ— should fail if an element is not supposed to exist but is found #integrationSuite[39m
      [31m- [39m[31mExpected 'Error: .element-does-exist was found when it should not exist!
    at onTimeout (/tmp/lambda_protractor/app/selection.ts:52:13)
    at Object.polledWait (/tmp/lambda_protractor/app/polled-wait.ts:35:9)
    at _getElements (/tmp/lambda_protractor/app/selection.ts:80:10)
    at Object.assertElementDoesNotExist (/tmp/lambda_protractor/app/selection.ts:157:20)
    at ElementFinderSync.assertElementDoesNotExist (/tmp/lambda_protractor/app/element-finder-sync.ts:38:12)
    at /tmp/lambda_protractor/test/spec/jasmine_reporter_test.ts:70:18
    at /tmp/lambda_protractor/test/spec/jasmine_reporter_test.ts:43:13
    at fiberContents (/tmp/lambda_protractor/node_modules/asyncblock/lib/asyncblock.js:92:26)
=== Pre-asyncblock stack ===
Error
    at module.exports (/tmp/lambda_protractor/node_modules/asyncblock/lib/asyncblock.js:29:15)
    at UserContext.<anonymous> (/tmp/lambda_protractor/test/spec/jasmine_reporter_test.ts:42:9)
    at /tmp/lambda_protractor/node_modules/jasminewd2/index.js:108:15
    at new ManagedPromise (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
    at ControlFlow.promise (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
    at schedulerExecute (/tmp/lambda_protractor/node_modules/jasminewd2/index.js:95:18)
    at TaskQueue.execute_ (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
    at asyncRun (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:2974:25)
    at /tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:668:7' to be undefined.[39m
          at FiberFlow.errorCallback (/tmp/lambda_protractor/test/spec/jasmine_reporter_test.ts:48:62)
          at FiberFlow.Flow._errorHandler (/tmp/lambda_protractor/node_modules/asyncblock-generators/lib/flow.js:530:18)
          at fiberContents (/tmp/lambda_protractor/node_modules/asyncblock/lib/asyncblock.js:98:18)

**************************************************
*                    Failures                    *
**************************************************

1) Protractor sync test cases should fail if an element is not supposed to exist but is found #integrationSuite
  [31m- [39m[31mExpected 'Error: .element-does-exist was found when it should not exist!
    at onTimeout (/tmp/lambda_protractor/app/selection.ts:52:13)
    at Object.polledWait (/tmp/lambda_protractor/app/polled-wait.ts:35:9)
    at _getElements (/tmp/lambda_protractor/app/selection.ts:80:10)
    at Object.assertElementDoesNotExist (/tmp/lambda_protractor/app/selection.ts:157:20)
    at ElementFinderSync.assertElementDoesNotExist (/tmp/lambda_protractor/app/element-finder-sync.ts:38:12)
    at /tmp/lambda_protractor/test/spec/jasmine_reporter_test.ts:70:18
    at /tmp/lambda_protractor/test/spec/jasmine_reporter_test.ts:43:13
    at fiberContents (/tmp/lambda_protractor/node_modules/asyncblock/lib/asyncblock.js:92:26)
=== Pre-asyncblock stack ===
Error
    at module.exports (/tmp/lambda_protractor/node_modules/asyncblock/lib/asyncblock.js:29:15)
    at UserContext.<anonymous> (/tmp/lambda_protractor/test/spec/jasmine_reporter_test.ts:42:9)
    at /tmp/lambda_protractor/node_modules/jasminewd2/index.js:108:15
    at new ManagedPromise (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
    at ControlFlow.promise (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
    at schedulerExecute (/tmp/lambda_protractor/node_modules/jasminewd2/index.js:95:18)
    at TaskQueue.execute_ (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
    at asyncRun (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:2974:25)
    at /tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:668:7' to be undefined.[39m
      at FiberFlow.errorCallback (/tmp/lambda_protractor/test/spec/jasmine_reporter_test.ts:48:62)
      at FiberFlow.Flow._errorHandler (/tmp/lambda_protractor/node_modules/asyncblock-generators/lib/flow.js:530:18)
      at fiberContents (/tmp/lambda_protractor/node_modules/asyncblock/lib/asyncblock.js:98:18)

Executed 1 of 5 specs [31m(1 FAILED) [39m(4 SKIPPED) in 0.775 sec.
[18:45:23] I/launcher - 0 instance(s) of WebDriver still running
[18:45:23] I/launcher - chrome #01 failed 1 test(s)
[18:45:23] I/launcher - overall: 1 failed spec(s)
[18:45:23] E/launcher - Process exited with error code 1
